README.md

```diff
@@ -1,19 +1,31 @@
-## Lambda Handler for Probot
+## AWS Lambda Extension for Probot
+A [Probot](https://github.com/probot/probot) extension to make it easier to run your Probot Apps in AWS Lambda.
 
-
-
-<details>
-<summary>TODO: Not published yet. Coming soon.</summary>
+## Usage
 
 ```shell
 $ npm install @probot/serverless-lambda
 ```
 
 ```javascript
 # handler.js
-const serverless = require('@probot/serverless-lambda');
-const plugin = require('./')
-module.exports.probot = serverless(plugin)
+const serverless = require('@probot/serverless-lambda')
+const appFn = require('./')
+module.exports.probot = serverless(appFn)
 ```
 
-</details>
+## Configuration
+This package moves the functionality of `probot run` into a handler suitable for usage on AWS Lambda + API Gateway. Follow the documentation on [Environment Configuration](https://probot.github.io/docs/configuration/) to setup your app's environment variables. You can add these to `.env`, but for security reasons you may want to use the [AWS CLI](https://aws.amazon.com/cli/) or [Serverless Framework](https://github.com/serverless/serverless) to set Environment Variables for the function so you don't have to include any secrets in the deployed package.
+
+For the private key, since AWS environment variables cannot be multiline strings, you could [Base64 encode](https://nodejs.org/api/buffer.html#buffer_buffers_and_character_encodings) the `.pem` file you get from the GitHub App or use [KMS](https://aws.amazon.com/kms/) to encrypt and store the key.
+
+## Differences from `probot run`
+
+#### Local Development
+Since Lambda functions do not start a normal node process, the best way we've found to test this out locally is to use [`serverless-offline`](https://github.com/dherault/serverless-offline). This plugin for the serverless framework emulates AWS Lambda and API Gateway on your local machine, allowing you to continue working from `https://localhost:3000/probot` before deploying your function to AWS.
```

Comment:
How to you actually use this plugin? Is there anything needed that's specific to probot?